src/Fixer/ClassNotation/SingleClassPropertyDeclarationFixer.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\ClassNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * Fixer for rules defined in PSR2 ¶4.2.
+ *
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+final class SingleClassPropertyDeclarationFixer extends AbstractFixer
```

Comment:
We have the `SingleImportPerStatementFixer`. So I think this fixer should be named `SingleClassPropertyPerStatementFixer`.